llama_index/llms/nvidia_tensorrt.py

```diff
@@ -0,0 +1,356 @@
+import gc
+import json
+import os
+import time
+import uuid
+from pathlib import Path
+from typing import Any, Callable, Dict, Optional, Sequence
+
+import numpy as np
+
+from llama_index.bridge.pydantic import Field, PrivateAttr
+from llama_index.callbacks import CallbackManager
+from llama_index.constants import DEFAULT_CONTEXT_WINDOW, DEFAULT_NUM_OUTPUTS
+from llama_index.llms.base import (
+    ChatMessage,
+    ChatResponse,
+    CompletionResponse,
+    LLMMetadata,
+    llm_chat_callback,
+    llm_completion_callback,
+)
+from llama_index.llms.custom import CustomLLM
+from llama_index.llms.generic_utils import completion_response_to_chat_response
+from llama_index.llms.generic_utils import (
+    messages_to_prompt as generic_messages_to_prompt,
+)
+
+EOS_TOKEN = 2
+PAD_TOKEN = 2
```

Comment:
I'm assuming these are specific to llama models? Are llama-based models the only ones supported?

Reply:
Yes, at this time it is. Others are coming in future releases. I figured when we bumped the `tensorrt_llm` version in a PR could also adjust this